sources/api/migration/migrator/src/main.rs

```diff
@@ -82,11 +85,37 @@ fn run() -> Result<()> {
             process::exit(0);
         });
 
-    let migrations = find_migrations(
-        &args.migration_directories,
-        &current_version,
-        &args.migrate_to_version,
-    )?;
+    // Prepare to load the locally cached tough repository to obtain the manifest.
+    let tough_datastore = args.working_directory.join(TOUGH_DATASTORE);
+    fs::create_dir_all(&tough_datastore).context(error::CreateDirectory {
+        path: &tough_datastore,
+    })?;
+    let metadata_url = dir_url(&args.metadata_directory)?;
+    let migrations_url = dir_url(&args.migration_directory)?;
+
+    // Failure to load the tough repo at the expected location is a serious issue because updog
+    // should always create a tough repo that contains at least the manifest, even if there are no
+    // migrations.
+    let repo = tough::Repository::load(
+        &tough::FilesystemTransport {},
+        tough::Settings {
+            root: File::open(&args.root_path).context(error::OpenRoot {
+                path: args.root_path.clone(),
+            })?,
+            datastore: &tough_datastore,
+            metadata_base_url: metadata_url.as_str(),
+            targets_base_url: migrations_url.as_str(),
+            limits: Default::default(),
```

Comment:
Good catch, I had specifically created these settings in the shared `update_metadata` and forgot to use them here. The author of `updog` and spelled them out instead of using default the version I'll push next does the same, just that both get them from a single source in `update_metadata`.